.github/workflows/ci.yaml

```diff
@@ -227,15 +227,15 @@ jobs:
           if [[ "${{ matrix.protocol }}" == "rpc" ]]; then
             just rpc-tests
           else
-            just slt-bin -v --protocol=${{ matrix.protocol }} 'sqllogictests/*'
+            just slt-bin --protocol=${{ matrix.protocol }} 'sqllogictests/*'
```

Comment:
this branch enables debug logging if you rerun the job in debug mode. we should keep this.